### PR TITLE
Add Android integration

### PR DESCRIPTION
**Pull request by joshliebe**
https://app.asana.com/0/0/1202939051866215/f

### CODE REVIEW

File: scripts/inject.mjs [hunk 1]
```diff
@@ -50,6 +50,8 @@ async function init() {
         initOther('inject/mozilla.js', process.argv[2]);
     } else if (process.argv[2] == "apple") {
         initOther('inject/apple.js', process.argv[2]);
+    } else if (process.argv[2] == "android") {
+        initOther('inject/android.js', process.argv[2]);
```
Comment (jonathanKingston) [inline scripts/inject.mjs:54]: You'll need another change in https://github.com/duckduckgo/content-scope-scripts/blob/main/package.json#L9 to output this file and then check in the resulting built file also :)

Comment (joshliebe) [reply scripts/inject.mjs:54]: Yeah, I just realized this. Will add 👍

**Comment by joshliebe** [comment]
@jonathanKingston I've updated the build script and added the built JS file. Please re-check!

**Review by jonathanKingston — APPROVED** [review]
LGTM thanks!

**Pull request merged**